modules/gitops-release-notes-1-3-6.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-3-6_{context}"]
+= Release notes for {gitops-title} 1.3.6
+
+{gitops-title} 1.3.6 is now available on {product-title} 4.7, 4.8, 4.9, and 4.10.
+
+[id="fixed-issues-1-3-6_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* In {gitops-title} improper access control allows admin privilege escalation link:https://access.redhat.com/security/cve/CVE-2022-1025[(CVE-2022-1025)]. This update fixes the issue.
```

Comment:
Comma before 'improper' @Srivaralakshmi  has also pointed this out. Please add it.